homeassistant/components/usb/__init__.py

```diff
@@ -102,6 +103,10 @@ async def _async_start_monitor(self) -> None:
         """Start monitoring hardware with pyudev."""
         if not sys.platform.startswith("linux"):
             return
+        info = await system_info.async_get_system_info(self.hass)
+        if info.get("docker") and not info.get("hassio"):
```

Comment:
Should we check the user as well, or will this work as expected with a non-root user in docker?

Reply:
```python3
from pyudev import Context, Monitor, MonitorObserver
context = Context()
monitor = Monitor.from_netlink(context)
monitor.filter_by(subsystem='input')
def print_device_event(device):
    print('background event {0.action}: {0.device_path}'.format(device))
observer = MonitorObserver(monitor, callback=print_device_event, name='monitor-observer')
observer.daemon
observer.start()
```
```
bash-5.1# su nobody -s /bin/bash
bash-5.1$ python3 run.py 
```

Appears to work as a user